lib/js/components/Buttons/IconButton/IconButton.consts.ts

```diff
@@ -29,3 +28,10 @@ export const ICON_BUTTON_COLORS = {
 	WARNING: 'warning',
 	INFO: 'info',
 } as const;
```

Comment:
> after this change story for IconButton won't work correctly. I see multiple usages of BUTTON_COLORS.MINOR there
by Damian J.

Reply:
In this PR there is no `minor` word in `components/Buttons` so it's outdated comment I think.